createDb.py

```diff
@@ -0,0 +1,48 @@
+#!/usr/bin/python3
+"""Module to create a database"""
+
+
+import sqlite3
+
+
+def createDB(path=None):
+    """Function to create a databases with required tables
+    path: Optional argument to already created database
+    """
+
+    """ Forcing foreign key constraints """
+
+    if path is None:
+        connection = sqlite3.connect('./AmbientDJ_DB.sqlite3')
+    else:
+        connection = sqlite3.connect(path)
```

Comment:
If the default value of `path` is `./AmbientDJ_DB.sqlite3`, you can save 3 lines.